database/migrations/2023_11_27_215150_create_wizard_templates_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWizardTemplatesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('wizard_templates', function (Blueprint $table) {
+            $table->id();
+            $table->uuid('uuid')->unique();
+            $table->unsignedBigInteger('process_template_id')->nullable();
+            $table->unsignedInteger('process_id');
+            $table->unsignedInteger('media_id');
```

Comment:
Hey @eiresendez I made changes to the migration, please don't forget to merge those. 
[FOUR-12173](https://processmaker.atlassian.net/browse/FOUR-12173)

Reply:
Done! I have pulled the latest changes